### ISSUE TITLE
Still active

### ISSUE THREAD

**Issue by t-bullock** [jdaly101/kassia#1]
Is this still being worked on? I might be interested in doing something similar, maybe like a cross-platform version of Pandouris.

**Comment by jdaly101** [comment]
Unfortunately I haven't been able to devote any time to it in the past year. After implementing a greedy line-breaking algorithm and successfully outputting neumes + text, I came to the conclusion that it desperately needed a good front end to create files that the script could parse.

**Comment by t-bullock** [comment]
Maybe I could contribute somehow, if you're still interested in the project. I'm not an expert in python, but I'm tired of dealing with the floating text boxes and all the hassle required to create a score using Word. What would you use to create the front end?

**Comment by jdaly101** [comment]
Yeah, I got tired of trying to write scores in Word myself. My inclination would be to create a web front end to create the input to a Python script which produces the output file(s). I'm pretty comfortable with JavaScript, but I'm open to other options, though. Feel free to email me at johnmdaly at gmail.

**Comment by jdaly101** [comment]
I split the project into a backend and a frontend. If you're still interested in collaborating, let me know.
